<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stretch_Dialog</class>
 <widget class="QDialog" name="Stretch_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Stretch Options</string>
  </property>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>9</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Define stretch by</string>
   </property>
   <widget class="QRadioButton" name="_100">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>23</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Coordinate entry        </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_101">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>44</y>
      <width>157</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Two points                 </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>213</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delta Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>185</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delta X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delta Z</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>98</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Define entities to stretch by</string>
   </property>
   <widget class="QRadioButton" name="_105">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>23</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Current selection      </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_106">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>44</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Window                    </string>
    </property>
   </widget>
  </widget>
  <widget class="RCDoubleSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>182</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_104">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>238</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
